collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```
@@ -106,6 +110,14 @@ private Config loadConfig(Map<String, Object> yamlConfig) throws MalformedObject
           yamlConfig = new HashMap<String, Object>();
         }
 
+        if (yamlConfig.containsKey("startDelay")) {
+          try{
+            cfg.startDelay = (Integer) yamlConfig.get("startDelay");
+          }
+          catch (NumberFormatException e) {
+            cfg.startDelay = 0;
```

We should throw an exception here